docs/architecture/adr-081-protobuf-mgmt.md

```diff
@@ -166,16 +165,15 @@ We currently aim to:
 ### Positive
 
 - We will still offer Go stub generation, proto linting and breakage checking.
-- Breakage checking will only happen on minor/patch releases to increase
+- Breakage checking will only happen on minor/patch releases to increase the
```

Comment:
FWIW, I think it's fine if we _run_ the check on all branches, as long as we don't mark it as required for submission on master.

Reply:
Cool, I've added a note in #8003 as a reminder.